devel/Dockerfile

```diff
@@ -35,6 +40,18 @@ RUN git clone --recursive https://github.com/phoenix-rtos/phoenix-rtos-project.g
     cd .. && rm -rf phoenix-rtos-project && \
     chmod -R a+rwX /opt/phoenix/utils
 
+# install qemu from xilinx
+RUN git clone https://github.com/Xilinx/qemu.git && \
+    cd qemu && \
+    git checkout tags/xilinx_v${DEVEL_XILINX_PLO_VERSION} && \
+    git submodule update --init dtc && \
+    mkdir build && cd build && \
+    ../configure --target-list="aarch64-softmmu" --enable-fdt --disable-kvm --disable-xen --enable-gcrypt --bindir="/usr/bin" --prefix="/usr" && \
+    make && \
+    strip --strip-unneeded /qemu/build/aarch64-softmmu/qemu-system-aarch64 && \
+    cp /qemu/build/aarch64-softmmu/qemu-system-aarch64 /usr/bin && \
```

Comment:
I would not assume we're at `/` when running this command (it can easily be changed with `WORKDIR` Dockerfile entry) - I would go for relative paths here as it won't hurt anybody :)

Reply:
Ok, wrong assumption from my side. Fixed.